apps/mobile-wallet/src/components/toasts/Toast.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2018 - 2023 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { colord } from 'colord'
+import { AlertCircle, CheckCircle, Icon, InfoIcon } from 'lucide-react-native'
+import {} from 'react-native'
+import { ToastConfigParams, ToastType } from 'react-native-toast-message'
+import { useTheme } from 'styled-components'
```

Comment:
shouldn't this be imported from `styled-components/native` and added in the line below?